app/scripts/modules/core/securityGroup/securityGroup.read.service.js

```diff
@@ -169,6 +169,7 @@ module.exports = angular.module('spinnaker.core.securityGroup.read.service', [
           }
         }
       }
+      data = _.uniq(data);
```

Comment:
this _should_ be getting handled by the `filter` calls on #L164 and #L166. Not sure why it isn't...